glean-core/src/util.rs

```diff
@@ -111,6 +111,52 @@ pub(crate) fn truncate_string_at_boundary_with_error<S: Into<String>>(
     }
 }
 
+#[cfg(all(target_arch = "x86", target_os = "windows"))]
+pub mod floating_point_context {
+    use libc::size_t;
+
+    #[link(name = "m")]
+    extern "C" {
+        fn _controlfp_s(current: *mut size_t, new: size_t, mask: size_t) -> size_t;
+    }
+
+    const MCW_PC: size_t = 0x00030000;
```

Comment:
nit: can we add some comments about these flags or link to the articles were we found them?